apps/styleguide/src/tables/RowEditModal.vue

```diff
@@ -269,12 +269,7 @@ export default {
         let defaultValue = {};
         this.tableMetadata.columns.forEach((column) => {
           // skip key in case of clone and visible
-          if (
-            data[column.id] &&
-            (!this.clone ||
-              column.key != 1 ||
-              !this.visibleColumns.includes(column.name))
-          ) {
+          if (data[column.id] && (!this.clone || column.key != 1)) {
```

Comment:
what is the meaning of column.key and what is the meaning of the magic '1' value ?

Reply:
See user guide: https://molgenis.github.io/molgenis-emx2/#/molgenis/use_schema?id=key

key indicates whether column is part of a unique constraint. Key=1 means primary key.
In other words: columns that are part of primary key should not be cloned because must be unique.

Reply:
in that case replace the 1 with meaningful constant PRIMARY_KEY , and the reason for not including the primary key could be added in a comment line

Reply:
that comment indeed exists on line 271. Because that already exists you don't see in track changes. I improve the comment for those who don't know yet that key=1 means primary key. I don't see added value of meaningful constant (adding extra code, unnecessary obfuscation) because in schema manual we also simply use '1'.